src/telemetry/telemetry.c

```diff
@@ -581,26 +674,12 @@ ts_telemetry_main(const char *host, const char *path, const char *service)
 	return false;
 }
 
-TS_FUNCTION_INFO_V1(ts_get_telemetry_report);
```

Comment:
isn't this function `ts_get_telemetry_report ` outward visible? Should we be changing it?